packages/__tests__/src/assert.ts

```diff
@@ -1,4 +1,4 @@
-import * as o from 'ospec';
```

Comment:
To get `esbuild` building the same as `ncc` I enabled `esModuleInteropt` which removes the need for import * from a lot of libs.